### PR TITLE
Setup common tests between remote and local

### PR DESCRIPTION
**Pull request by tateexon**
Sets up the tests to run on both local and remote to verify in both places in CI.
Covers all the test cases in TT-150 but the chaos test which I will add in another PR.

### CODE REVIEW

File: .github/workflows/e2e.yaml [hunk 1]
```diff
@@ -32,6 +32,12 @@ jobs:
           file: Dockerfile.base
           AWS_REGION: ${{ secrets.QA_AWS_REGION }}
           AWS_ROLE_TO_ASSUME: ${{ secrets.QA_AWS_ROLE_TO_ASSUME }}
+      - name: Base Image Built
+        run: |
+          echo "### chainlink image used for this test run :link:" >>$GITHUB_STEP_SUMMARY
+          echo "\`${{ env.CHAINLINK_VERSION }}\`" >>$GITHUB_STEP_SUMMARY
+          echo "### test-base-image image tag for this test run :ship:" >>$GITHUB_STEP_SUMMARY
+          echo "\`ci.${{ github.sha }}\`" >>$GITHUB_STEP_SUMMARY
```
Comment (AnieeG) [inline .github/workflows/e2e.yaml:40]: Can we include the same for chainlink integration-tests workflow as well?

File: e2e/common/test_common.go [hunk 2]
```diff
@@ -107,7 +124,7 @@ func TestConnectWithoutManifest(t *testing.T) {
 
 func Test5NodesSoakEnvironmentWithPVCs(t *testing.T) {
 	t.Parallel()
-	testEnvConfig := getTestEnvConfig(t)
+	testEnvConfig := GetTestEnvConfig(t)
 	e := presets.EVMSoak(testEnvConfig)
 	err := e.Run()
```
Comment (AnieeG) [inline e2e/common/test_common.go:129]: better to include an err check here as well

File: presets/presets.go [hunk 3]
```diff
@@ -71,8 +71,9 @@ func EVMReorg(config *environment.Config) *environment.Environment {
 				},
 			},
 		})).
-		AddHelm(chainlink.New(0, map[string]interface{}{
+		AddHelm(chainlink.NewVersioned(0, "0.0.11", map[string]interface{}{
```
Comment (AnieeG) [inline presets/presets.go:74]: Any specific reason for this version?

File: .github/workflows/e2e.yaml (hunk 1 above)
Comment (tateexon) [reply .github/workflows/e2e.yaml:40]: Already exists for smoke, soak, and benchmark tests in chainlink!

File: presets/presets.go (hunk 3 above)
Comment (tateexon) [reply presets/presets.go:74]: As far as I know this was the last helm chart version that accepted non toml configs.

**Review by kalverra — APPROVED** [review]

**Pull request merged**